spec/mock-request.js

```diff
@@ -59,21 +75,24 @@ HttpBackend.prototype = {
                     console.log("  [%s] Flushed assigned amount: %s", path, numToFlush);
                     defer.resolve();
                 } else {
-                    setTimeout(tryFlush, 0);
+                    // setImmediate(tryFlush);
+                    process.nextTick(tryFlush);
+                    // tick(0);
```

Comment:
Though one wonders why `setImmediate` is used below and `process.nextTick` is used here? Any reason for it? If not, can we be consistent please.